inference/core/models/roboflow.py

```diff
@@ -713,7 +715,12 @@ def __init__(
             self.validate_model()
         except ModelArtefactError as e:
             logger.error(f"Unable to validate model artifacts, clearing cache: {e}")
-            self.clear_cache()
+            if DISK_CACHE_CLEANUP == False:
+                self.clear_cache(
+                    delete_from_disk=random.random() < 0.01
```

Comment:
this is in order to make it try to redownload the model with 1% chance when there is error in case the download was the issue / got corrupted?

Reply:
wait no it was always doing it before, so its meant to do it less often now if DISK_CACHE_CLEANUP == False

Reply:
Yea, and also, this wont change any of the functionality for other users of inference because DISK_CACHE_CLEANUP = True for everyone else by default (as was always the case).

DISK_CACHE_CLEANUP = False is applicable to serverless - where usually we dont want valid models to be deleted from the shared disk cache we have introduced.